changes.md

```diff
@@ -0,0 +1,101 @@
+# OCI Distribution Specification Changes
+
+This doc should be updated when changes are made to the specification, indicating what is different.
```

Comment:
These are not intended to be _only_ breaking changes, but _all_ changes?

Reply:
What do you think about saying "when non-trivial changes are made" or "With the exception of de-minimis modifications, this doc should be updated when changes are made.."

Reply:
The changes must include clarifications because any ambiguity may cause two implementations to differ in a way that the clarification breaks one and not the other. For example `Clarified expected behavior response to manifest HEAD request.` did cause breakages although one could have argued that the `HTTP` spec was already clear about the behavior of `HEAD` making the change trivial. We just need to be careful about what we define as "trivial" to account for this.

Reply:
kk then de-minimis (such as spelling changes)